Dockerfile-noble

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:24.04
+
+RUN apt-get -qq update \
+    && DEBIAN_FRONTEND=noninteractive apt-get -qq -y install \
+        apt-transport-https \
+        autoconf \
+        build-essential \
+        cmake \
+        curl \
+        jq \
+        debhelper \
+        default-jre-headless \
+        devscripts \
+        dh-autoreconf \
+        git \
+        libgmp-dev \
+        libsodium-dev \
+        libssl-dev \
+        ncurses-dev \
+        unzip \
+        libwxgtk3.2-dev \
+        libsctp1 \
+        libgflags-dev \
+        libsnappy-dev \
+        zlib1g-dev \
+        libbz2-dev \
+        liblz4-dev \
+        libzstd-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+ARG BUILD_OTP_VERSION="26.2.1"
+ENV OTP_VERSION=$BUILD_OTP_VERSION
+RUN OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-${OTP_VERSION}.tar.gz" \
+    && curl -fsSL -o otp-src.tar.gz "$OTP_DOWNLOAD_URL" \
+    && mkdir otp-src \
+    && tar -zxf otp-src.tar.gz -C otp-src --strip-components=1 \
+    && cd otp-src \
+    && export ERL_TOP=`pwd` \
+    && ./otp_build autoconf && ./configure && make -j$(nproc) && make install
+
+ENV PORTABLE=1
+ENV ROCKSDB_VERSION=8.11.4
```

Comment:
This version is different/newer than the other images, I'm a bit concerned about this diversion